WalletWasabi/Stores/IndexStore.cs

```diff
@@ -148,18 +148,37 @@ private async Task TryEnsureBackwardsCompatibilityAsync()
 			try
 			{
 				// Before Wasabi 1.1.5
-				var oldIndexFilepath = Path.Combine(EnvironmentHelpers.GetDataDir(Path.Combine("WalletWasabi", "Client")), $"Index{Network}.dat");
+				var oldIndexFilePath = Path.Combine(EnvironmentHelpers.GetDataDir(Path.Combine("WalletWasabi", "Client")), $"Index{Network}.dat");
+
+				// Before Wasabi 1.1.6
+				var oldFileNames = new[] {
+					"ImmatureIndex.dat" ,
+					"ImmatureIndex.dat.dig",
+					"MatureIndex.dat",
+					"MatureIndex.dat.dig"
+				};
+				var oldIndexFolderPath = Path.Combine(EnvironmentHelpers.GetDataDir(Path.Combine("WalletWasabi", "Client")), "BitcoinStore", Network.ToString());
+
+				foreach (var fileName in oldFileNames)
+				{
+					var oldFilePath = Path.Combine(oldIndexFolderPath, fileName);
+					if (File.Exists(oldFilePath))
+					{
+						File.Copy(oldFilePath, oldFilePath.Replace(oldIndexFolderPath, WorkFolderPath), true);
+						File.Delete(oldFilePath);
```

Comment:
Q: what's the difference between `Copy` + `Delete`  and `Move`?

Reply:
Move doesn't overwrite, but you're right, it should be more efficient using Move.